_translator-files/po/de/navigation.po

```diff
@@ -78,7 +80,7 @@ msgstr "/wiki/Software-Manual"
 #: ../wiki/en/misc/navigation.yml:0
 #, no-wrap
 msgid "FAQ"
-msgstr "FAQ"
+msgstr "FAQ - Häufig gestellte Fragen"
```

Comment:
Please leave it like that. It will probably look strange in the navigation.

Reply:
Changed to FAQ